### PR TITLE
Added the version where the lifecycle-hooks were introduced.

### PR DESCRIPTION
**Pull request by KwintenP**
Added the version where the lifecycle-hooks were introduced.

### CODE REVIEW

File: _posts/2016-03-29-exploring-angular-1.5-lifecycle-hooks.md [hunk 1]
```diff
@@ -36,6 +36,8 @@ Lifecycle hooks in Angular landed first in the Angular 2 alpha release and they
 
 However, this article is on Angular 1.5. Since Angular 1 is evolving in a way to keep the gap to Angular 2 as small as possible, some lifecycle callbacks have been backported to the current "best Angular yet". Let's take a look at them one by one.
 
+Note: The Angular lifecycle-hooks weren't introduced until version 1.5.3.
```
Comment (PascalPrecht) [inline _posts/2016-03-29-exploring-angular-1.5-lifecycle-hooks.md]: Can we make this rather "positive" by saying it has been introduce in 1.5.3 ? :)

Comment (KwintenP) [inline _posts/2016-03-29-exploring-angular-1.5-lifecycle-hooks.md]: Updated ;)

File: _posts/2016-03-29-exploring-angular-1.5-lifecycle-hooks.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ Lifecycle hooks in Angular landed first in the Angular 2 alpha release and they
 
 However, this article is on Angular 1.5. Since Angular 1 is evolving in a way to keep the gap to Angular 2 as small as possible, some lifecycle callbacks have been backported to the current "best Angular yet". Let's take a look at them one by one.
 
-Note: The Angular lifecycle-hooks weren't introduced until version 1.5.3.
+Note: The Angular lifecycle-hooks were introduced in version 1.5.3.
```
Comment (PascalPrecht) [inline _posts/2016-03-29-exploring-angular-1.5-lifecycle-hooks.md]: Awesome! Now make it bold using `**`

**Comment by PascalPrecht** [comment]
Left one comment. Can you squash your commits? Will merge once done :)

**Comment by KwintenP** [comment]
Done ;).

**Comment by PascalPrecht** [comment]
Perrrrrfect! Thank you!

**Pull request merged**